lib/models/release.ts

```diff
@@ -389,6 +389,53 @@ const getReleaseModel = function (
 		);
 	}
 
+	/**
+	 * @summary Finalizes a draft release
+	 * @name finalize
+	 * @public
+	 * @function
+	 * @memberof balena.models.release
+	 *
+	 * @param {String|Number} commitOrId - release commit (string) or id (number)
+	 * @fulfil {void}
+	 * @returns {Promise}
+	 *
+	 * @example
+	 * balena.models.release.finalize(123).then(function() {
+	 * 	console.log('finalized!');
+	 * });
+	 *
+	 * @example
+	 * balena.models.release.finalize('7cf02a6').then(function() {
+	 * 	console.log('finalized!');
+	 * });
+	 *
+	 */
+	async function finalize(commitOrId: string | number): Promise<void> {
+		if (commitOrId == null) {
+			throw new errors.BalenaReleaseNotFound(commitOrId);
+		}
+
+		if (isId(commitOrId)) {
+			await pine.patch<Release>({
+				resource: 'release',
+				id: commitOrId,
+				body: {
+					is_final: true,
+				},
+			});
+		} else {
+			const { id } = await get(commitOrId, { $select: 'id' });
+			await pine.patch<Release>({
+				resource: 'release',
+				id,
+				body: {
+					is_final: true,
+				},
+			});
+		}
```

Comment:
@toochevere what I meant was actually:
```suggestion
		const { id } = await get(commitOrId, { $select: 'id' });
		await pine.patch<Release>({
			resource: 'release',
			id: commitOrId,
			body: {
				is_final: true,
			},
		});
```

Reply:
oh, of course